http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -391,7 +394,11 @@ public void onContent(Response response, ByteBuffer content)
             }
         };
 
-        RequestInfo requestInfo = RequestInfo.from(jettyRequest, System.currentTimeMillis());
+        long requestTimestamp = System.currentTimeMillis();
```

Comment:
System.currentTimeMillis() is sensitive to clock adjustments. We may want to use System.nanoTime() if all we're doing is measure durations.

Reply:
Other than the request timestamp field we use other fields to measure durations, so I will update them to use `nanoTime()`.